docs/systems/jax/ippo.md

```diff
@@ -1,3 +1,3 @@
 > 🚧 **Note:** This only applies to the callback redesign of Mava.
```

Comment:
There are notes like this throughout the docs. We can remove all of them since we will now only have Jax systems.

Reply:
Cool. Removed :+1:

Reply:
Thank you :)